src/server/api.ts

```diff
@@ -121,6 +121,14 @@ export const configureRouter = ({
     const game = games.find((g) => g.name === gameName);
     if (!game) ctx.throw(404, 'Game ' + gameName + ' not found');
 
+    if (
+      numPlayers != null &&
+      ((game.minPlayers != null && numPlayers < game.minPlayers) ||
+        (game.maxPlayers != null && numPlayers > game.maxPlayers))
```

Comment:
Could we avoid the loose inequality operator here? I find it can be hard to reason about all the different ways operands are compared.

I'd propose treating `numPlayers` more carefully because it's user input. We don't need to be so careful about the game config so I'd just check `minPlayers`/`maxPlayers` is truthy. Perhaps something like:

```js
    if (
      numPlayers !== undefined &&
        (typeof numPlayers !== 'number' ||
          (game.minPlayers && numPlayers < game.minPlayers) ||
          (game.maxPlayers && numPlayers > game.maxPlayers))
    ) {
```

That way we also throw an error if `numPlayers` is not actually a number.

Reply:
Sounds good. Since `Number.parseInt()` is used in the definition of `numPlayers` above and will always be a `number` (but could be `NaN`) I ended up using:
```javascript
ctx.request.body.numPlayers !== undefined && (Number.isNaN(numPlayers || ...
```